### PR TITLE
add adjustable parscale and parscale multiplier

### PR DESCRIPTION
**Pull request by adamjorr**

### CODE REVIEW

File: utils/optimize_parameters.R [hunk 1]
```diff
@@ -1,4 +1,23 @@
 #! Rscript --vanilla
+#
+# Copyright (c) 2014-2018 Reed A. Cartwright
```
Comment (reedacartwright) [inline utils/optimize_parameters.R]: 2017-2018 Me
2018 Adam Orr

**Comment by adamjorr** [comment]
Is that correct?

**Comment by reedacartwright** [comment]
I'd add "Copyright (c)" at the beginning of your line.

--
Reed A. Cartwright, PhD
Barrett Honors Faculty
Assistant Professor of Genomics, Evolution, and Bioinformatics
School of Life Sciences
Human and Comparative Genomics Laboratory
The Biodesign Institute
Arizona State University
==================
Availability: http://links.asu.edu/CartwrightCalendar
Address: The Biodesign Institute, PO Box 875301, Tempe, AZ 85287-5301 USA
Packages: The Biodesign Institute, 1001 S. McAllister Ave, Tempe, AZ
85287-5301 USA
Office: Biodesign B-220C, 1-480-965-9949
Website: http://cartwrig.ht/

**Comment by adamjorr** [comment]
:+1:

**Pull request merged**